lldb/include/lldb/lldb-enumerations.h

```diff
@@ -1305,6 +1305,14 @@ enum CompletionType {
   eTerminatorCompletion = (1ul << 27)
 };
 
+/// Specifies if children need to be re-computed
+/// after a call to \ref SyntheticChildrenFrontEnd::Update.
+enum class ChildCacheState {
+  eDynamic = 0, ///< Children need to be recomputed dynamically.
+  eConstant =
+      1, ///< Children will never change and don't need to be recomputed.
```

Comment:
Is `eConstant` really constant? In other words, if `Update()` returns `eConstant` the first time, will it always return `eConstant` every other invocation? If not, then I would like to see a different name/names. Do you find issue with the verb approach I suggested?